### ISSUE TITLE
Is there any way I can give the uploadURL path dynamically?

### ISSUE THREAD

**Issue by dipens** [sunhater/kcfinder#109]
For example, if I am using KCFinder in a textbox I would like to pass the upload directory as a parameter of the url which will be created inside dynamically inside the uploadURL folder. 

Explanation. Suppose my uploadURL is currently set to `TAPL/W3TrillSoftFiles`  Is there any way where I can supply an extra parameter in this url `/kcfinder/browse.php?type=files&dir=files/public` for example like `/kcfinder/browse.php?type=files&dir=files/public&subDir=PO/200846` and it will create the PO directory and 200846 directory on the fly inside the parent directory dynamically and set this directory as the root directory in kcfinder UI? So the new root directory becomes `TAPL/W3TrillSoftFiles/PO/200846/`. I hope I am clear enough with this. And thanks for making kcfinder.

**Comment by noncent** [comment]
Yes, You can use KCFinder SESSION Key 'KCFINDER' to pass your dynamic config settings:

e.g. 
```php
// Set KcFinder session array to setup upload url
$_SESSION['KCFINDER'] = array 
(
	'uploadURL' => 'http://www.yourdomain.com/upload',
	'uploadDir' => 'var/etc/www/project/upload'
);
```
Link : **[KcFinder_CkEditor_Absolute_Url](https://github.com/neerajsinghsonu/KcFinder_CkEditor_Absolute_Url)**
